platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -1390,13 +1395,7 @@ private Response continueLogout(
         entityServiceInfo =
             getServiceProvidersMap().get(entityId).getLogoutService(incomingBinding);
```

Comment:
It looks like `getLogoutService` can return null. We should probably attempt to do something if this occurs or at least check it before we dereference it on the next line.

Reply:
Should I also keep the SonarQube finding I deleted?
```
if (entityServiceInfo == null) {	
     LOGGER.info("Could not find entity service info for {}", entityId);	
     return continueLogout(logoutState, cookie, incomingBinding);	
}
```

Reply:
No I don't think we want that code. A log message at most.